hermes-api/build.gradle

```diff
@@ -7,6 +7,7 @@ dependencies {
     compile group: 'javax.ws.rs', name: 'javax.ws.rs-api', version: '2.0'
     compile group: 'com.fasterxml.jackson.core', name: 'jackson-annotations', version: versions.jackson
     compile group: 'com.fasterxml.jackson.jaxrs', name: 'jackson-jaxrs-json-provider', version: versions.jackson
+    compile group: 'com.fasterxml.jackson.datatype', name: 'jackson-datatype-jsr310', version: versions.jackson
```

Comment:
I see you have only added one error code to this module, so I am wondering why have you added this dep? Do we need it here? I am asking because hermes-api should be as light as possible :)